python/tests/test_aac/context/test_language_error.py

```diff
@@ -0,0 +1,16 @@
+from unittest import TestCase
+
+from aac.context.language_context import LanguageContext
+from aac.context.language_error import LanguageError
+
+class TestLanguageError(TestCase):
+    def test_language_error(self):
```

Comment:
With having the `location` set as an optional, should also test that it handles fine receiving `None`, if we don't decide to move away from that implementation.

Reply:
This can be ignored since we are going to try and move away from having it be an optional and going to force some kind of location provision, even if a notice of something like "No location"